### PR TITLE
Add ca file for npm

### PR DESCRIPTION
**Pull request by tianfeng92**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ async function setUpNpmConfig (registry, strictSSL) {
     rollback: false,
     fund: false,
     'strict-ssl': strictSSL,
-    noproxy: 'registry.npmjs.org'
+    noproxy: 'registry.npmjs.org',
+    cafile: process.env.CA_FILE
```
Comment (vrunoa) [inline src/utils.js:54]: @tianfeng92 are there any side effect if this env var is not defined?

Comment (tianfeng92) [reply src/utils.js:54]: tested in docker mode. no side effect when signing `undefined` to cafile. Let me add more check before this line

Comment (tianfeng92) [reply src/utils.js:54]: no.. actually it doesn't work. investigating

Comment (tianfeng92) [reply src/utils.js]: works.

**Review by alexplischke — APPROVED** [review]

File: src/utils.js [hunk 2]
```diff
@@ -50,8 +50,13 @@ async function setUpNpmConfig (registry, strictSSL) {
     rollback: false,
     fund: false,
     'strict-ssl': strictSSL,
-    noproxy: 'registry.npmjs.org'
-  });
+    noproxy: 'registry.npmjs.org',
+  };
+  if (process.env.CA_FILE) {
+    config.cafile = process.env.CA_FILE;
```
Comment (alexplischke) [inline src/utils.js:56]: You could technically do a shorthand version of it, no?
`cafile` is by default null (according to https://docs.npmjs.com/cli/v7/using-npm/config). So you could do `cafile: process.env.CA_FILE | null` up above.

Comment (tianfeng92) [reply src/utils.js:56]: perfect! let me try this.

**Pull request merged**